src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -11,81 +11,83 @@
+                    name="creator" />
+                </div>
             </div>
-          </div>
-          <div class="form-group">
-            <label class="col-sm-1 control-label" for="wi-detail-assignee">Assignee</label>
-            <div class="col-sm-5">
-              <input type="text" id="wi-detail-assignee" placeholder="Assignee" class="form-control"
-                [(ngModel)]="workItem.fields['system.assignee']" 
-                name="assignee" />
+            <div class="form-group">
+                <label class="col-sm-1 control-label" for="wi-detail-assignee">Assignee</label>
+                <div class="col-sm-5">
+                <input type="text" id="wi-detail-assignee" placeholder="Assignee" class="form-control"
+                    [(ngModel)]="workItem.fields['system.assignee']" 
+                    name="assignee" />
+                </div>
             </div>
-          </div>
-          <div class="form-group">
-            <label class="col-sm-1 control-label" for="wi-detail-state">State</label>
-            <div class="col-sm-5">
-              <select id="wi-detail-state" class="form-control"
-                [(ngModel)]="workItem.fields['system.state']" 
-                name="states">
-                <option *ngFor="let state of workItemStates" [value]="state">{{state}}</option>
-              </select>
+            <div class="form-group">
+                <label class="col-sm-1 control-label" for="wi-detail-state">State</label>
+                <div class="col-sm-5">
+                <select id="wi-detail-state" class="form-control"
+                    [(ngModel)]="workItem.fields['system.state']" 
+                    name="states">
+                    <option *ngFor="let state of workItemStates" [value]="state">{{state}}</option>
+                </select>
+                </div>
             </div>
-          </div>
-          <div class="form-group">
-            <div class="col-sm-offset-1 col-sm-5">
-              <button type="submit" class="btn btn-primary" id="workItemDetail_btn_save" 
-                [disabled]="!almWorkItemDetailForm.form.valid || !(workItem.fields['system.title'] | almTrim)">
-                Save
-              </button>
-              <button type="button" id="workItemDetail_btn_back" class="btn btn-default"        
-                (click)="goBack(almWorkItemDetailForm.form.dirty)">Back</button>
+            <div class="form-group">
+                <div class="col-sm-offset-1 col-sm-5">
+                <button type="submit" class="btn btn-primary" id="workItemDetail_btn_save" 
+                    [disabled]="!almWorkItemDetailForm.form.valid || !(workItem.fields['system.title'] | almTrim)">
+                    Save
+                </button>
+                <button type="button" id="workItemDetail_btn_back" class="btn btn-default"        
+                    (click)="goBack(almWorkItemDetailForm.form.dirty)">Back</button>
```

Comment:
Back button needs to be enabled so that the user can navigate back to the list page.